sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -59,16 +91,16 @@ export interface DeleteRepositoryOptions extends OperationOptions {
 
 // @public
 export interface DeleteRepositoryResult {
-    deletedRegistryArtifactDigests?: string[];
-    deletedTags?: string[];
+    readonly deletedManifests?: string[];
+    readonly deletedTags?: string[];
```

Comment:
would be nice to default these arrays to empty array

Reply:
Fixed.